src/components/RightPanelContents.tsx

```diff
@@ -50,16 +50,13 @@ const RightPanelContents = (props: AllProps) => {
     setSearchQuery(e.target.value);
   };
 
-  const handleSubmit = () => {
+  const handleSubmit = (e: React.FormEvent) => {
     console.log(searchQuery);
     searchByContent({ searchQuery });
+    e.preventDefault();
   };
 
-  const handleKeyDown = (e: React.KeyboardEvent) => {
-    if (e.key === "Enter") {
-      handleSubmit();
-    }
-  };
+  const [searchButtonIsHovered, setSearchButtonIsHovered] = useState(false);
```

Comment:
Would it make sense to use the `:hover` CSS selector instead?

Reply:
I agree that it's a bit ugly.... I didn't use the `:hover` CSS selector because when the button is hovered over, I want the `button` element *and* its child `svg` to display "hovered" styled. I also tried two different `:hover` selectors (one in each element), but that didn't work since they have different bounding boxes and therefore don't apply the hovered styles at the same time. If you have a better solution, I'm definitely open to it!

Reply:
I was wrong about not being able to use the CSS `:hover` selector... I realize now that I could just put the `:hover` selector on the parent component and have it set the styles for the child component.